routes/index.js

```diff
@@ -563,7 +572,7 @@ router.put('/discover', function(req, res, next) {
     //apply 'show songs in' filter
     var loadMore = function(songs2d, done) {
         if (langShown !== 'all') {
-            songs2d = songs2d.filter((s) => s.lang === langShown)
+            songs2d = songs2d.filter((s) => s.lang == langShown)
```

Comment:
Did you see my comment on line 550? I think we should be consistent and just explicitly use toString().